classes_ja.js

```diff
@@ -0,0 +1,50 @@
+//////  1  /////////////
+
+class Human {
+  constructor(name, age) {
+    this.name = name;
+    this.age = age;
+  }
+  ageOneYear() {
+    return this.age++;
+  }
+  eatin() {
+    return `I ${this.name} Pahh, dah, paah pah pahhh, I'm lovin' it....`;
+  }
+}
+
+let pancho = new Human("Panchito", 35);
+
```

Comment:
Nitpick, but the eating function asks you to log the result. You should console.log within the function instead of returning the string.